.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Is this necessary? Can you please remove this from the tracked files if it is not necessary?

Reply:
This is just a small file to help you not push unnecessary folders and files, i think so

Reply:
This file came from gitignore, no big deal